xpk.py

```diff
@@ -1107,6 +1328,8 @@ def run_gke_cluster_create_command(args) -> int:
       f' --cluster-version={args.gke_version} --location-policy=BALANCED'
       f' --machine-type={machine_type}'
       ' --scopes=storage-full,gke-default'
+      ' --enable-ip-alias '
+      f' --create-subnetwork name={args.cluster}-subnetwork '
```

Comment:
Some customers use shared VPC. Does XPK allow a user to select an option to use shared VPC? If yes, would this subnetwork also be created in the shared VPC? cc @Obliviour

Reply:
I intend to use this https://github.com/google/xpk/blob/main/xpk.py#L2877 variable and test. I have not tested shared VPC so far. Thanks for the resources provided offline!

Reply:
Will test shared VPC once requirements are clearer, perhaps part of the next PR.